docs/src/_documentation/how_tos/05-integrate-cablecar.md

```diff
@@ -38,12 +38,12 @@ mrujs.start({
 Now, any link or form with `data-method="<method>"` or `data-remote="true"` will have the following `Accept` header:
 
 ```js
-"application/vnd.cable-ready.json, */*"
+"text/vnd.cable-ready.json, */*"
```

Comment:
Maybe it might be worth to also mention that you can override the MIME-Type by configuring it like:
```js

mrujs.start({
  plugins: [
    new CableCar(CableReady, { mimeType: "text/vnd.cable-ready.json" })
  ]
})
```

Reply:
I've added a commit for that: 7ff8e1d then. Thanks for the suggestion.